app/models/user.rb

```diff
@@ -1,6 +1,20 @@
 class User < ApplicationRecord
   has_many :votes
+  has_many :works
   has_many :ranked_works, through: :votes, source: :work
 
   validates :username, uniqueness: true, presence: true
+
+  def work_owner(work)
+    self.works.find_by(id: work)
+  end
```

Comment:
The name is a little confusing here.  Maybe instead name this method, `owner?(work)`